tests/integ/test_dataframe.py

```diff
@@ -3440,3 +3440,29 @@ def test_drop_columns_special_names(session):
         Utils.check_answer(df2, [Row(1), Row(2)])
     finally:
         Utils.drop_table(session, table_name)
+
+
+def test_dataframe_interval_operation(session):
```

Comment:
@sfc-gh-nkrishna Please add a test in this file for this new expression.

Reply:
@sfc-gh-yixie You mean a test for Interval(day=2) actually corresponding to `INTERVAL 2 DAY`?

Reply:
@sfc-gh-yixie INTERVAL isn't really a column, so how would you suggest I test this here?

Reply:
@sfc-gh-yixie How would you suggest I add a test for Interval here since it's just a keyword not a column?

Reply:
Like in your test, you can compose a column by using Interval.
```python
df = df.select(df["a"] + Column(Interval(day=2))
```
Then assert the column name is the same as the name returned from the sql.